src/concurrency/cpu_affinity.rs

```diff
@@ -29,21 +29,32 @@ impl CpuAffinityMask {
         this
     }
 
+    pub fn chunk_size(target: &rustc_target::spec::Target) -> usize {
+        // The actual representation of the CpuAffinityMask is [c_ulong; _], in practice either
+        //
+        // - [u32; 32] on 32-bit platforms
+        // - [u64; 16] everywhere else
+        match target.pointer_width {
```

Comment:
Isn't this just `pointer_width.bytes()`?

Then the natural return type becomes `u64` which will also work better in sched_getaffinity.

Reply:
Oh sorry, it's not a `Size`.

`pointer_width * 8` then.

Please also add a FIXME to look up the actual size of `core::ffi::c_ulong` in the future.